src/views/RecordingsView.vue

```diff
@@ -379,15 +379,34 @@ export default {
         const tag = tags[i];
         const tagName = tag.animal === null ? tag.event : tag.animal;
         const taggerId = tag.taggerId;
-
         this.addToListOfTags(tagItems, tagName, tag.automatic, taggerId);
       }
 
       if (tracks) {
         for (let j = 0; j < tracks.length; j++) {
           const track = tracks[j];
-          for (let i = 0; i < track.TrackTags.length; i++) {
-            const tag = track.TrackTags[i];
+          // For track tags, pick the best one, which is the "master AI" tag.
```

Comment:
Note that now we will only show ONE ai tag for all recordings to everyone.   It will be harder to see how much our AI is diverging in its guesses.